src/test/java/org/broadinstitute/consent/http/models/InstitutionDomainMapTest.java

```diff
@@ -0,0 +1,83 @@
+package org.broadinstitute.consent.http.models;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.google.gson.Gson;
+import java.util.Map;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+
+public class InstitutionDomainMapTest {
```

Comment:
`public` is unnecessary for class and member functions in JUnit 5
```suggestion
class InstitutionDomainMapTest {
```